README.md

```diff
@@ -1,7 +1,18 @@
 # The Active Admin Store
-[![Travis CI](https://img.shields.io/travis/activeadmin/activeadmin/master.svg)           ](https://travis-ci.org/activeadmin/activeadmin)
-[![Dependency Status](https://gemnasium.com/badges/github.com/activeadmin/demo.activeadmin.info.svg)](https://gemnasium.com/github.com/activeadmin/demo.activeadmin.info)
+[![Travis CI](https://img.shields.io/travis/activeadmin/activeadmin/master.svg)](https://travis-ci.org/activeadmin/activeadmin)
 
 This is a test application to demo Active Admin.
 
 http://demo.activeadmin.info
+
+
+## Local development setup
+
+Dependencies: Ruby 2.5+ and PostgreSQL
+
+1. Clone project using Git
+2. Run `$ bundle install`
+3. Change Postgres username/password in `config/database.yml` (it will be good if this user has right to create DB)
```

Comment:
I would prefer that the user doesn't need to edit any files. We can add an extra optional steps for the case where the user does not have a permissive postgreSQL configuration (where user `postgres` and empty password does not work). Like:

* Create the DB user with `sudo -u postgres psql -c "CREATE USER demo_aa WITH CREATEDB NOCREATEROLE PASSWORD 'demo_aa'"`
* Export the environment to set the credentials with `export DATABASE_USERNAME=demo_aa` and `export DATABASE_PASSWORD=demo_aa`. You can use [direnv](https://github.com/direnv/direnv) so that you don't need to export these environment variables every time.

Reply:
Cool solution. It works for me.

So I think it would be OK to use it, despite this works only for Debian-like distro.
Many developers use OSX. The SQLite was universal solution (I don't remember if SQLite is installed by default in OSX).

Reply:
Yeah, true. I don't use `OSX` but I thought the default configuration would just work there. If not, maybe someone can contribute instructions for that OS.